tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -369,11 +369,16 @@ def assign_csp_resources(self, argin):
         argout = []
         dish = {}
         try:
+            print ("CSP Assign resource input:",argin )
```

Comment:
Remove print statements from this block of code.

Reply:
Done.